tools/Localizations/Sources/Localizations/ExportTask.swift

```diff
@@ -67,6 +61,30 @@ struct ExportTask {
         try! task.run()
         task.waitUntilExit()
     }
+    
+    // Get Locales that are in the Xcode project
+    private func getProjectLocales() -> (Array<String>) {
+    
+        var localesList:[String] = []
+
+        let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: xcodeProjPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path)
+        let filePaths = blockzillaFolder?.allObjects as! [String]
```

Comment:
Although in this case probably safe, it is best to avoid forced casts (`as!`). An `if let` here can make sure that both blockzillaFolder and filePaths are setup properly. Or go a `guard`:

```
guard if let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: xcodeProjPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path),
    let filePaths = blockzillaFolder?.allObjects as? [String]
     else {
    return []
}
```

The first is so long, I would move it into a computer property or a `getBlockzillaFolder` function.